packages/cli/scripts/build.mjs

```diff
@@ -16,6 +16,7 @@ const publicApiEnabled = process.env.N8N_PUBLIC_API_DISABLED !== 'true';
 
 generateUserManagementEmailTemplates();
 generateTimezoneData();
+generateThirdPartyLicenses();
```

Comment:
I don't think we need to run this on every local build, as it slows down the build during development. We could instead have a placeholder file e.g. in the `cli` root and only overwrite in the `<reporoot>/scripts/build-n8n.mjs`